config/config.go

```diff
@@ -13,7 +13,7 @@ type Config struct {
 	Main `mapstructure:",squash"`
 
 	Pastel *pastel.Config `mapstructure:"pastel" json:"pastel,omitempty"`
-	Nats   *nats.Config   `mapstructure:"nats" json:"nats,omitempty"`
+	Nats   *nats.Config   `mapstructure:"chat" json:"chat,omitempty"`
```

Comment:
We are going to use `nats` connection not only for chatting. I mean, there will be only one `nats` connection for all services, so everything what relates to nats connection should be called `nats`, if you have some specific settings exactly for `chat`, create for it a new struct.